### ISSUE TITLE
Waterfox won't open

### ISSUE THREAD

**Issue by nn901140** [MrAlex94/Waterfox-Old#89]

**Comment by aeiouaeiouaeiouaeiouaeiouaeiou** [comment]
According to the version in mozglue you are using a [modern version of Waterfox](https://github.com/WaterfoxCo/Waterfox).
Do you have the [latest version](https://aka.ms/vs/17/release/vc_redist.x64.exe) of the Visual C++ library installed?
What kind of processor is used in the computer? The browser now only works on systems [with Sandy Bridge or newer](https://github.com/WaterfoxCo/Waterfox/issues/2558).

**Comment by Squall-Leonhart** [comment]
unpatched Windows 8.1 and windows 7 not running ESU are both unable to run waterfox with e10s enabled,

**Comment by Squall-Leonhart** [comment]
>  What kind of processor is used in the computer? The browser now only works on systems with Sandy Bridge or newer.

False, Nahalem is the first intel chip with 4.2 support, you only need Sandy for AVX (which Waterfox doesn't use)

**Comment by timur-g** [comment]
I think that issues for WF and Classic are separated, for new WF like this one link is https://github.com/WaterfoxCo/Waterfox/issues and this here is for Classic issues.
